codecov.yml

```diff
@@ -1,6 +1,6 @@
 ---
 ignore:
-  - testsuite/*
+  - "testsuite"
```

Comment:
no need to add quotes here

Reply:
I am just following - https://docs.codecov.com/docs/ignoring-paths. Are you sure we don't need quotes?